### PR TITLE
Match Dewford

### PR DESCRIPTION
**Pull request by PokeCodec**
Credit goes to @Kaz for this one. I also added an enum for height and weight

### CODE REVIEW

**Comment by PokeCodec** [comment]
I hate this code

File: src/battle_script_commands.c [hunk 1]
```diff
@@ -4650,8 +4650,7 @@ static void Cmd_switchinanim(void)
 
 static void Cmd_jumpifcantswitch(void)
 {
-    s32 i;
-    s32 lastMonId;
+    s32 i, lastMonId;
```
Comment (PikalaxALT) [inline src/battle_script_commands.c:4653]: Revert or finish matching

File: src/berry_crush.c [hunk 2]
```diff
@@ -1986,29 +1997,29 @@ static u32 BerryCrushCommand_BeginNormalPaletteFade(struct BerryCrushGame *game,
...
+	selectedPals[0] |= selectedPals[1];
+	selectedPals[1] = (u32)params[3];
+	selectedPals[1] <<= 24;
+
+	selectedPals[0] |= selectedPals[1];
+	params[0] = params[9];
+
+	color = params[8];
+	color <<= 8;
+	color |= params[7];
```
Comment (PikalaxALT) [inline src/berry_crush.c:2019]: Fix indentation

File: src/dewford_trend.c [hunk 3]
```diff
@@ -219,20 +219,37 @@ void BufferTrendyPhraseString(void)
     ConvertEasyChatWordsToString(gStringVar1, s->words, 2, 1);
 }
 
+#ifndef NONMATCHING
```
Comment (PikalaxALT) [inline src/dewford_trend.c:222]: No.

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

**Comment by PokeCodec** [comment]
Done

**Review by PikalaxALT — APPROVED** [review]

**Pull request merged**